boardswarm-cli/src/main.rs

```diff
@@ -389,7 +389,7 @@ struct ConfigureArg {
 
 #[derive(Debug, Subcommand)]
 enum Command {
-    /// Configure boardswarm instances
+    /// Configure boardswarm instances access
```

Comment:
The purpose of the change is exactly to avoid tricking the user into believing they configure server instances with boardswarm-cli. Instances are configured via corresponding server.conf file.

Reply:
naming is hard; the configure subcommand naming was probably a bad choice in hindsight for example the reason @andrzejtp  points out. Given it's still so early i don't mind changing it around though as for sure there is a bunch of refactoring we can do to make it more user then developer friendly :)

Looking at some other tools
*  `az` the azure cli tool uses `az login` for similar functionality 
* `glab` the gitlab cli tool uses `glab auth login` with other subcommands to see the status
* `osc`  wants to setup login if it doesn't recognize the passed remote, but doesn't have a dedicate subcommand
* `doctl` has `doctl auth init`  with other subcommands and options to list and create different "context" for multiple accounts

Interestingly `az` also has configure ` configure: Manage Azure CLI configuration. This command is interactive.`  

Overall the current `configure` subcommand is really about setting up authentication to (multiple) remote boardswarm servers and being slightly overloaded to also be able to change e.g. the default. Overall the `doctl` approach seems likely the best with a specific `auth` subcommand to handle this area, with `auth` subcommands to e.g. list, switch (change default), remove, re-login/update

Reply:
I quite like the idea changing the argument name to `auth` | `authentication` and having subcommands like `login`. I can see why `configure` would confuse people into thinking it configures the server and devices :-)